llvm/test/CodeGen/AIE/aie2/schedule/implicit_ops.mir

```diff
@@ -57,10 +57,8 @@ alignment:       16
 body:             |
   bb.0.entry:
     ; CHECK-LABEL: name: implicit_WAW
-    ; CHECK: BUNDLE implicit-def $wl0, implicit-def $x0, implicit-def $wh0, implicit killed $p0, implicit killed $p1 {
-    ; CHECK-NEXT:   $wl0 = VLDA_dmw_lda_w_ag_idx_imm killed $p0, 0, implicit-def $x0 :: (load (<8 x s32>) from stack - 64)
-    ; CHECK-NEXT:   $wh0 = VLDB_dmw_ldb_ag_idx_imm killed $p1, 0 :: (load (<8 x s32>) from stack - 32)
-    ; CHECK-NEXT: }
+    ; CHECK: $wl0 = VLDA_dmw_lda_w_ag_idx_imm killed $p0, 0, implicit-def $x0 :: (load (<8 x s32>) from stack - 64)
+    ; CHECK-NEXT: $wh0 = VLDB_dmw_ldb_ag_idx_imm killed $p1, 0 :: (load (<8 x s32>) from stack - 32)
```

Comment:
That test now changed meaning, can you add addresspace info to the MMOs so both loads can actually inssue in parallel?

Reply:
Yes.